packetbeat/docs/fields.asciidoc

```diff
@@ -6,7 +6,7 @@ This file is generated! See etc/fields.yml and scripts/generate_field_docs.py
 [[exported-fields]]
 == Exported Fields
 
-This document describes the fields that are exported by Docs/Fields.Asciidoc. They are
+This document describes the fields that are exported by Packetbeat. They are
```

Comment:
@andrewkroh I remember seeing that change from you just recently. That is what I got when I run `make update`.